movement/movement.c

```diff
@@ -384,6 +384,14 @@ void app_init(void) {
     movement_state.settings.bit.to_interval = MOVEMENT_DEFAULT_TIMEOUT_INTERVAL;
     movement_state.settings.bit.le_interval = MOVEMENT_DEFAULT_LOW_ENERGY_INTERVAL;
     movement_state.settings.bit.led_duration = MOVEMENT_DEFAULT_LED_DURATION;
+#ifdef MAKEFILE_TIMEZONE
+    for (int i = 0, count = sizeof(movement_timezone_offsets) / sizeof(movement_timezone_offsets[0]); i < count; i++) {
+        if (movement_timezone_offsets[i] == MAKEFILE_TIMEZONE) {
+            movement_state.settings.bit.time_zone = i;
+            break;
+        }
+    }
```

Comment:
I feel like there should be a way to set this statically, without looping over the time zones at runtime. We need to figure out the index of the timezone. Perhaps the makefile can figure out or call a script that figures it out, then it can pass the index via preprocessor definition.